dev/com.ibm.ws.session/resources/com/ibm/ws/session/resources/WASSessionCore.nlsprops

```diff
@@ -131,4 +131,6 @@ CommonMessage.miscData=Miscellaneous data: {0}
 # -------------------------------------------------------------------------------------------------
 SessionContext.invalidCloneSeparator=SESN0800W: Property CloneSeparator should be exactly one char and it cannot be space. The current value of CloneSeparator specified is "{0}"
 SessionContext.invalidCloneSeparator.explanation=The property CloneSeparator should be exactly one char and it cannot be space.
-SessionContext.invalidCloneSeparator.useraction=Modify the CloneSeparator custom property.
\ No newline at end of file
+SessionContext.invalidCloneSeparator.useraction=Modify the CloneSeparator custom property.
+# -------------------------------------------------------------------------------------------------
+# SESN0802E in use in com.ibm.ws.session.cache/resources/com/ibm/ws/session/cache/resources/WASSessionCache.nlsprops
```

Comment:
Claim a range for session cache instead of a single message id.  Otherwise we will need to update this fine every single time we add a message, and it will become a mess

Reply:
I thought about doing that, but I couldn't find the range that sessions had generally claimed to reserve a subset of that, although I suppose I could claim a range regardless.